WrightTools/artists/_interact.py

```diff
@@ -50,6 +56,20 @@ def _at_dict(data, sliders, xaxis, yaxis):
     }
 
 
+def create_local_global_radio(ax, local):
+    if mpl.__version__ >= "3.7":
```

Comment:
```suggestion
    if mpl.__version_info__ >= (3, 7):
```

Better check, less reliant on string version numbers (e.g.  mpl "3.10" would fail your condition)